alma/lib/Model/CarrierData.php

```diff
@@ -25,6 +25,7 @@
 namespace Alma\PrestaShop\Model;
 
 use Carrier;
```

Comment:
Do you use Carrier ?

Reply:
@Francois-Gomis You're right 😅 I remove its for the fix